src/main/java/hudson/plugins/s3/S3Profile.java

```diff
@@ -296,6 +301,6 @@ public String toString() {
     }
 
     private ProxyConfiguration getProxy() {
-        return Jenkins.getActiveInstance().proxy;
+        return Jenkins.get().proxy;
```

Comment:
I understand it's a `getActiveInstance()` to `get()` method switch, but accessing the `proxy` field is not really clean. 
Don't we have a cleaner way to get the `ProxyConfiguration`?

Reply:
Not that I'm aware of. Neither the Jenkins nor the ProxyConfiguration classes have getters for it.

Reply:
then it's fine for me.